### PR TITLE
Use the reserved domains example.com and example.org

### PR DESCRIPTION
**Pull request by javiereguiluz**
| Q | A |
| --- | --- |
| Doc fix? | yes |
| New docs? | no |
| Applies to | all |
| Fixed tickets | - |

Most of the current docs already use the reserved domains, but I think it's OK to add this recommendation to the docs standards.

### CODE REVIEW

**Comment by stof** [comment]
:+1:

File: contributing/documentation/standards.rst [hunk 1]
```diff
@@ -57,6 +57,8 @@ Code Examples
   Unless the example requires a custom bundle, make sure to always use the
   ``AppBundle`` bundle to store your code;
 * Use ``Acme`` when the code requires a vendor name;
+* Use ``example.com`` as the domain of sample URLs and ``example.org`` when a
+  second domain is required. Both domains are `reserved by the IANA`_.
```
Comment (WouterJ) [inline contributing/documentation/standards.rst]: I wouldn't add an order in the standard, just what domains you can use: example.com, example.org and example.net

I would also link to the RFC instead: http://tools.ietf.org/html/rfc2606#section-3

Comment (javiereguiluz) [inline contributing/documentation/standards.rst]: I've updated the link to the RFC. However, I don't understand your first suggestion. This document is about standards, so everything is "an order" (i.e. "strong suggestion") to the doc contributor.

Comment (javiereguiluz) [inline contributing/documentation/standards.rst]: Sorry for my previous comment. I deleted it because I didn't understand your proposal.

**Comment by xabbuh** [comment]
:+1:

**Comment by weaverryan** [comment]
Awesome - thanks!

**Pull request merged**